spec/features/user_adds_bar_spec.rb

```diff
@@ -0,0 +1,164 @@
+require "rails_helper"
```

Comment:
This includes stories beyond `user adds bar`. Please split this up into multiple files - one for each feature (and titled accordingly)